src/chart.zoom.js

```diff
@@ -159,7 +172,7 @@ function doPan(chartInstance, deltaX, deltaY) {
 
 		helpers.each(chartInstance.scales, function(scale, id) {
 			if (scale.isHorizontal() && directionEnabled(panMode, 'x') && deltaX !== 0) {
-				panScale(scale, deltaX);
+				panScale(scale, deltaX, helpers.getValueOrDefault(chartInstance.options, defaultOptions));
```

Comment:
Noticed that this wasn't picking up the default pan speed if there wasn't one in the user options.
doPan() now sets panOptions.speed  to either the user or default value before passing panOptions to panScale